src/Components/AdvancedSearch/Internal/AdvancedSearchResultDetailsList/AdvancedSearchResultDetailsList.types.ts

```diff
@@ -1,30 +1,21 @@
-import {
-    IColumn,
-    IStyle,
-    IStyleFunctionOrObject,
-    ITheme
-} from '@fluentui/react';
+import { IStyle, IStyleFunctionOrObject, ITheme } from '@fluentui/react';
 import {
     IADTTwin,
-    IPropertyInspectorAdapter,
-    Locale,
-    Theme
+    IPropertyInspectorAdapter
 } from '../../../../Models/Constants';
 
 export interface IAdvancedSearchResultDetailsListProps {
-    filteredTwinsResult: IADTTwin[];
-    additionalColumns: IColumn[];
+    twins: IADTTwin[];
+    searchedProperties: string[]; //string of fieldnames, for specific properties that were used in the search
     // callback function from parent on what to do once the user selects or deselects a twin
-    callbackFunction?: void; //not meant to be optional, just did that for now so the stories work
+    OnTwinSelection?: (IADTTwin) => void; //not meant to be optional, just did that for now so the stories work
```

Comment:
let's make it required.

Reply:
it is but that's gonna be on the next PR.  Not needed for the goal of this PR